src/yamerl_constr.erl

```diff
@@ -195,6 +195,11 @@ new(Source) ->
 %% happens an exception is raised. When set to `true', the node is
 %% constructed as if it was a plain YAML node without any tag.</dd>
 %% <dd>Default: `false'.</dd>
+%% <dt>`{keep_duplicate_keys, boolean()}'</dt>
+%% <dd>Flag to keep duplicate keys in maps. By default all duplicate keys
+%% in maps/proplists will ignored and the last occurence of a key will
+%% prevail. If this flag is enabled all keys will remain. This flag only
+%% works when the `detailed_constr' flag is set to `true'.</dd>
```

Comment:
Duplicate keys could be kept for proplists-based simple constructions as well. Could you please add that?

Reply:
@dumbbell Sure! I've added them in my latest update.